common/src/main/java/org/apache/comet/parquet/Utils.java

```diff
@@ -22,18 +22,19 @@
 import org.apache.parquet.column.ColumnDescriptor;
 import org.apache.parquet.schema.LogicalTypeAnnotation;
 import org.apache.parquet.schema.PrimitiveType;
+import org.apache.spark.sql.internal.SQLConf;
 import org.apache.spark.sql.types.*;
 
+import org.apache.comet.CometConf;
+
 public class Utils {
   public static ColumnReader getColumnReader(
-      DataType type,
-      ColumnDescriptor descriptor,
-      int batchSize,
-      boolean useDecimal128,
-      boolean useLazyMaterialization) {
-    // TODO: support `useLegacyDateTimestamp` for Iceberg
+      DataType type, ColumnDescriptor descriptor, int batchSize, SQLConf conf) {
+    boolean useDecimal128 = (Boolean) CometConf.COMET_USE_DECIMAL_128().get(conf);
+    boolean useLazyMaterialization = (Boolean) CometConf.COMET_USE_LAZY_MATERIALIZATION().get(conf);
+    boolean useLegacyDateTimestamp = (Boolean) CometConf.COMET_USE_DECIMAL_128().get(conf);
```

Comment:
Just making sure... Is it correct to use `COMET_USE_DECIMAL_128` for `useLegacyDateTimestamp`?

Reply:
I meant to have `COMET_EXCEPTION_ON_LEGACY_DATE_TIMESTAMP`